courses/free-courses-pt_BR.md

```diff
@@ -209,6 +209,8 @@
 
 * [Curso de Javascript Completo](https://www.youtube.com/playlist?list=PL2Fdisxwzt_d590u3uad46W-kHA0PTjjw) - Programação Web
 * [Curso Grátis de JavaScript e ECMAScript para Iniciantes](https://www.youtube.com/playlist?list=PLHz_AreHm4dlsK3Nr9GVvXCbpQyHQl1o1) - Gustavo Guanabara (Curso em Vídeo)
+* [Curso Javascript | Interagindo com a DOM + Manipulando Arrays e Objetos](https://www.youtube.com/watch?v=QBDUYIHkF0M) - Fernanda Kipper
+* [Curso Javascript | Sintaxe básica (Tipos, Variáveis, Loops, Funções, Condicionais)](https://www.youtube.com/watch?v=mo3fpbW_eTA) - Fernanda Kipper
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes